usermgr/src/main/java/com/composum/sling/core/usermanagement/service/impl/AuthorizablesImpl.java

```diff
@@ -132,9 +132,9 @@ public <T> void build(final QueryBuilder<T> builder) {
                                 .nameMatches(StringUtils.isNotBlank(nameQueryPattern) ? nameQueryPattern : "%"));
                         builder.setSortOrder("@name", QueryBuilder.Direction.ASCENDING);
                         if (selector != null) {
-                            if (selector.isInstance(UserWrapper.class)) {
```

Comment:
Perhaps rather UserWrapper.class.isAssignableFrom(selector)? That leaves room for later subclasses and feels more in the spirit of the original instanceof . That'd apply to the other selector.equals occurrences in this file, too.